packages/authorization-process/src/model/domain/errors.ts

```diff
@@ -371,6 +372,16 @@ export function eserviceNotDelegableForClientAccess(
   });
 }
 
+export function purposeAdditionNotAllowedForClient(
```

Comment:
```suggestion
export function clientKindNotAllowed(
```
This could be generalized in order to be used in both operations (addition/removal)